<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="param_path" default="$(find-pkg-share autoware_simpl)/config/simpl.param.yaml"/>
  <arg name="data_path" default="$(find-pkg-share autoware_simpl)/data"/>
  <arg name="input/vector_map" default="/map/vector_map"/>
  <arg name="output/objects" default="/output/objects"/>
  <arg name="build_only" default="false"/>

  <node pkg="autoware_simpl" exec="autoware_simpl_ego_exe" output="screen">
    <param from="$(var param_path)" allow_substs="true"/>
    <remap from="~/input/vector_map" to="$(var input/vector_map)"/>
    <remap from="~/output/objects" to="$(var output/objects)"/>
    <param name="build_only" value="$(var build_only)"/>
  </node>
</launch>
